fsys_test.go

```diff
@@ -0,0 +1,135 @@
+package skeletonkit_test
+
+import (
+	"embed"
+	"flag"
+	"io"
+	"path/filepath"
+	"strings"
+	"testing"
+	"text/template"
+
+	"github.com/gostaticanalysis/skeletonkit"
+	"github.com/tenntenn/golden"
+)
+
+type appInfo struct {
+	Name       string
```

Comment:
unnecessary exporting

Reply:
For template data, I think the fields have to be exported.

Reply:
Oh, you are right.